app/scripts/controllers/onboarding.js

```diff
@@ -0,0 +1,39 @@
+const ObservableStore = require('obs-store')
+const extend = require('xtend')
+
+/**
+ * @typedef {Object} OnboardingOptions
+ * @property {Boolean} seedPhraseBackedUp Indicates whether the user has completed the seed phrase backup challenge
```

Comment:
This doesn't seem correct—if the `seedPhraseBackedUp` and `initState` properties are both on the same options object, wouldn't using it below in the extend generate an incorrect initial state? That is,

```js
const extend = require('xtend')
// => undefined
opts = {initState: {}, seedPhraseBackedUp: true}
// => { initState: {}, seedPhraseBackedUp: true }
extend({ seedPhraseBackedUp: null }, opts.initState)
// => { seedPhraseBackedUp: null }

```